### ISSUE TITLE
Official Kanboard plugin?

### ISSUE THREAD

**Issue by tamielbr** [EstebanMonge/kanboard_imap_tasks#9]
Hi - just wondering if this became an official plugin. It doesn't show inside Kanboard as an option.

**Comment by EstebanMonge** [comment]
Nope... Is poor programmed and unmaintained but functional plugin
